sample-app/src/pages/VerticalSearchPage.tsx

```diff
@@ -0,0 +1,107 @@
+import ResultsCount from '../components/ResultsCount';
+import AlternativeVerticals from '../components/AlternativeVerticals';
+import DecoratedAppliedFilters from '../components/DecoratedAppliedFilters';
+import StaticFilters from '../components/StaticFilters';
+import VerticalResults from '../components/VerticalResults';
+import SpellCheck from '../components/SpellCheck';
+import LocationBias from '../components/LocationBias';
+import Facets from '../components/Facets';
+import '../sass/VerticalSearchPage.scss';
+import { StandardCard } from '../components/cards/StandardCard';
+import { useLayoutEffect } from 'react';
+import { useAnswersActions } from '@yext/answers-headless-react';
+
+const staticFilterOptions = [
+  {
+    label: 'canada',
+    fieldId: 'c_employeeCountry',
+    value: 'Canada',
+  },
+  {
+    label: 'remote',
+    fieldId: 'c_employeeCountry',
+    value: 'Remote'
+  },
+  {
+    label: 'usa',
+    fieldId: 'c_employeeCountry',
+    value: 'United States',
+  },
+  {
+    label: 'tech',
+    fieldId: 'c_employeeDepartment',
+    value: 'Technology'
+  },
+  {
+    label: 'consult',
+    fieldId: 'c_employeeDepartment',
+    value: 'Consulting',
+  },
+  {
+    label: 'fin',
+    fieldId: 'c_employeeDepartment',
+    value: 'Finance',
+  }
+]
+
+const facetConfigs = {
+  c_employeeDepartment: {
+    label: 'Employee Department!'
+  }
+}
+
+export default function VerticalSearchPage(props: {
+  verticalKey: string
+}) {
+  const answersActions = useAnswersActions();
+  useLayoutEffect(() => {
```

Comment:
I believe useLayoutEffect is fired before DOM mutation? I think we don't need to wrap this around useLayoutEffect. I think something like this would work:
```export default function VerticalSearchPage(props: {
  verticalKey: string
}) {
  const answersActions = useAnswersActions();
  answersActions.setVerticalKey(props.verticalKey);
  answersActions.executeVerticalQuery();
  ...```